lib/sheet.js

```diff
@@ -16,13 +16,17 @@ const wrap = (target) =>
 const math = wrap(Math);
 
 const getValue = (target, prop) => {
-  if (prop === 'Math') return math;
-  const { expressions, data } = target;
-  if (data.has(prop)) return data.get(prop);
+  try {
+    if (prop === 'Math') return math;
+    const { expressions, data } = target;
+    if (data.has(prop)) return data.get(prop);
 
-  const expression = expressions.get(prop);
-  if (expression) return expression.compute();
-  return target.handleUnknownIdentifier(prop, target);
+    const expression = expressions.get(prop);
+    if (expression) return expression.compute();
+    return target.handleUnknownIdentifier(prop, target);
+  } catch (error) {
+    return target.emit('error', error, target);
+  }
```

Comment:
Very long code block inside `try/catch`, this may throw multiple error, so I'd propose to minimize `try` block to `compute` block or split it into a few try blocks